libraries/AP_TECS/AP_TECS.cpp

```diff
@@ -900,8 +915,8 @@ void AP_TECS::update_pitch_throttle(int32_t hgt_dem_cm,
                                     float load_factor)
 {
     // Calculate time in seconds since last update
-    uint32_t now = AP_HAL::micros();
-    _DT = MAX((now - _update_pitch_throttle_last_usec), 0U) * 1.0e-6f;
+    uint64_t now = AP_HAL::micros64();
```

Comment:
the MAX() didn't do anything, as subtracting two unsigned numbers always gives unsigned.
I think we should check for any 32 bit micros and ensure they are never used for anything long term. If TECS was not calls for 71 minutes then it could have wrapped and done the wrong thing.